packages/nvidia_nat_core/tests/nat/front_ends/auth_flow_handlers/test_websocket_flow_handler.py

```
@@ -223,3 +227,141 @@ async def create_websocket_message(self, msg):
     # Verify timeout RuntimeError is raised (demonstrates partial error handling)
     error_message = str(exc_info.value)
     assert "Authentication flow timed out" in error_message
+
+
+# ---------------------------------------------------------------------------
+# WebSocketMessageHandler – preflight authentication
```

Ruff (0.15.17)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)